roles/common/defaults/main.yml

```diff
@@ -26,6 +26,8 @@ _apt_packages_default:
   dbus: "{{ apt_package_state }}"
   git: "{{ apt_package_state }}"
   libnss-myhostname: "{{ apt_package_state }}"
+  python2.7-minimal: "{{ apt_package_state }}"
```

Comment:
Looks like we need to do the actual plain `python` package to get `/usr/bin/python` which is what Ansible needs.